src/plugin/plg_backend_ftps/index.go

```diff
@@ -2,8 +2,9 @@ package plg_backend_ftps
 
 import (
 	"fmt"
-	. "github.com/mickael-kerjean/filestash/server/common"
-	"github.com/secsy/goftp"
+	. "github.com/mickael-kerjean/filestash/src/common"
+	"github.com/BobCashStory/goftp"
```

Comment:
I'd rather keep the dependency on the original goftp

Reply:
i agreee but goftp has a bug for now and it doesn't allow ftps to work properly, i did a Pr on they repo https://github.com/secsy/goftp/pull/40